spreadsheet-weather-based-marketing/classes/config.gs

```diff
@@ -65,6 +65,8 @@ class Config {
       'col-lon':                'Longitude',
       'col-formula':            'Activation Formula',
       'col-last-updated':       'Last Updated',
+      'any-api-url':            'Api URL',
+      'any-api-headers':        'Api Headers',
```

Comment:
Can we just make weather api one instance of AnyApi (e.g. put the OpenWeather URL as default and remove the weather-specific code as much as possible?

Reply:
I think it makes sense in the future. At the moment "Weather API" part is more or less good tested. "Any API" is not that much tested, so I would do the refactoring after we have couple of projects done with the "Any API" approach.